README.md

```diff
@@ -37,5 +37,9 @@ Storybook is deployed to [Chromatic](https://chromatic.com) via `yarn chromatic`
 - CHAIN_ID = 1 // (or 5 for goerli)
 - NODE_URLS ={"1":"<https://mainnet.infura.io/v3/xxx","5":"https://goerli.infura.io/v3/xxx"}>
 - NEXT_PUBLIC_MAILCHIMP_URL="<https://umaproject.us19.list-manage.com/subscribe/post?u={user}&id={id}&f_id={f_id}>"
+- NEXT_PUBLIC_OVAL_MAILCHIMP_URL="<https://umaproject.us19.list-manage.com/subscribe/post?u={user}&id={id}&f_id={f_id}>"
 - NEXT_PUBLIC_DEFAULT_APY=30.5
-- NEXT_PUBLIC_TVS=83M
+- NEXT_PUBLIC_TVS=83M=  // Osnap TVS shown, use formatted string
+- NEXT_PUBLIC_OEV_LOST= // backup oev value if dune is not working, use raw number
+- DUNE_API_KEY= // for getting total OEV lost
+- AIRTABLE_API_KEY=  // for capturing osnap, oval leads
```

Comment:
just wanted to document some missing envs

Reply:
nice 👍